src/BaGet.AWS/BaGet.AWS.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+
+    <Description>The libraries to host BaGet on AWS.</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AWSSDK.S3" Version="3.3.30" />
+    <PackageReference Include="JetBrains.Annotations" Version="2018.2.1" />
```

Comment:
What's the JetBrains dependency for?

Reply:
One of the configuration extensions throws if the input is null, but Jetbrains Resharper from then on assumes that the input _can_ be null. Using the ContractAnnotation from Resharper allows me to explain to R# that if the input is null, it throws - hence it "can't be null". 

I'll remove it :)